### PR TITLE
Add stubs for text field types

### PR DESCRIPTION
**Pull request by ceesgeene**
fixes https://github.com/mglaman/phpstan-drupal/issues/360

### CODE REVIEW

**Comment by mglaman** [comment]
Hm, that's odd. I don't know why they're all returning `*ERROR*`

File: stubs/Drupal/text/Plugin/Field/FieldType/TextItem.php
Comment (mglaman) [inline stubs/Drupal/text/Plugin/Field/FieldType/TextItem.php:1]: Can we rename this to `.stub`?

File: stubs/Drupal/text/Plugin/Field/FieldType/TextItemBase.php
Comment (mglaman) [inline stubs/Drupal/text/Plugin/Field/FieldType/TextItemBase.php:1]: Can we rename this to `.stub`?

File: stubs/Drupal/text/Plugin/Field/FieldType/TextLongItem.php
Comment (mglaman) [inline stubs/Drupal/text/Plugin/Field/FieldType/TextLongItem.php:1]: Can we rename this to `.stub`?

File: stubs/Drupal/text/Plugin/Field/FieldType/TextWithSummaryItem.php
Comment (mglaman) [inline stubs/Drupal/text/Plugin/Field/FieldType/TextWithSummaryItem.php:1]: Can we rename this to `.stub`?

File: stubs/Drupal/text/Plugin/Field/FieldType/TextItem.php
Comment (ceesgeene) [reply stubs/Drupal/text/Plugin/Field/FieldType/TextItem.php:1]: Oops

**Comment by mglaman** [comment]
🙌 looks like renaming stub files fixed the error. 11.x-dev run is unrelated

**Review by mglaman — APPROVED** [review]

**Comment by mglaman** [comment]
Thanks!

**Pull request merged**